common/gpu/intel/opencl.nix

```diff
@@ -0,0 +1,14 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.hardware.intel.opencl.enable = lib.mkEnableOption (lib.mkDoc
```

Comment:
```suggestion
  options.hardware.intel.opencl.enable = lib.mkEnableOption (lib.mdDoc
```

Reply:
Also, the module name is `hardware.intelgpu`, not `hardware.intel`.